docs/sparkr.md

```diff
@@ -133,7 +133,7 @@ specifying `--packages` with `spark-submit` or `sparkR` commands, or if initiali
 
 <div data-lang="r" markdown="1">
 {% highlight r %}
-sparkR.session(sparkPackages = "com.databricks:spark-avro_2.11:3.0.0")
+sparkR.session()
```

Comment:
Eh, @dbtsai, I think you can just switch this to other datasources like `spark-redshift` or `spark-xml`, and fix the description above `you can find data source connectors for popular file formats like Avro`.

Reply:
I am not familiar with R. Can you elaborate? Thanks.

Reply:
oh, but the problem is other packages probably wouldn't have _2.12 distribution. hm, I think this can be left as was for now.

At least I am going to release spark-xml before Spark 3.0.0 anyway. I can try to include 2.12 distribution as well and fix it here later.

Reply:
I mean it's related with using external package, it looks so but Avro is kind of internal source now .. so it's out of date.

Reply:
Get you. 

BTW, are you familiar with Mima? I still can not figure out why it's still failing.

Reply:
I am not an expert but just know a bit. The mima change look right from a cursory look.

Reply:
Let me try to take a look as well this weekends.

Reply:
adding @JoshRosen

Reply:
I'd not worry about this example too much - this could be `com.databricks:spark-avro_2.12:3.0.0`

Reply:
I thought `com.databricks:spark-avro_2.12` is deprecated and no longer exist.

Reply:
It can be anything. Rather than make this an example that requires maintenance, why not just change the surrounding text to not necessarily refer to an Avro connector, and make this a dummy package like com.acme:spark-foo_2.12 ?

Reply:
yes!
(although, let's not use spark here - don't want to encourage naming packages with spark in the name)

Reply:
I just did a text search and replacement, and I didn't read the context of having `sparkPackages = "com.databricks:spark-avro_2.11:3.0.0"` here. My bad.

Although avro is now part of spark codebase, but it's in external package which is not in the classpath by default. How about I change it to `sparkPackages = "org.apache.spark:spark-avro_2.12:3.0.0"` here?

Reply:
That seems fine. A dummy package is also fine.